config/7.10.0/dashbuilder/rhpam-standalone-dashbuilder.yaml

```diff
@@ -47,6 +47,7 @@ dashbuilder:
               app: "[[.ApplicationName]]"
               application: "[[.ApplicationName]]"
               service: "[[.ApplicationName]]-[[.Dashbuilder.Name]]"
+              cluster: "jgrp.k8s.dashbuilder.[[.Dashbuilder.Name]]"
```

Comment:
in order to ensure cluster name uniqueness within a namespace, the label should probably look more like this -
`"jgrp.k8s.console.[[.ApplicationName]].[[. Dashbuilder.Name]]"`

Reply:
maybe we could just remove the console word from the name? 
wdyt @tchughesiv ?

Reply:
Console word?

Reply:
Oh, that’s my fault... I pasted from above. I’ll edit my comment

Reply:
I meant, instead to have:

```
jgrp.k8s.console.[[.ApplicationName]].[[.Console.Name]]
```

have just:

```
jgrp.k8s.[[.ApplicationName]].[[.Console.Name]]
```

without the component type

Reply:
agree

Reply:
Ok